### PR TITLE
make n_subj required

### PR DESCRIPTION
**Pull request by xiazhu**
small change to make n_subj a required parameter.

### CODE REVIEW

File: brainiak/factoranalysis/htfa.py [hunk 1]
```diff
@@ -64,23 +64,19 @@ class HTFA(TFA):
     Parameters
     ----------
 
-    R : list of 2D arrays, element i has shape=[n_voxel, n_dim]
```
Comment (mihaic) [inline brainiak/factoranalysis/htfa.py]: Was `R` not used?

File: brainiak/factoranalysis/htfa.py [hunk 2]
```diff
@@ -64,23 +64,19 @@ class HTFA(TFA):
...
-        Each element in the list contains the scanner coordinate matrix
-        of fMRI data of one subject.
-
-    K : int, default: 50
+    K : int
         Number of factors to compute.
 
+    n_subj : int, default: 2
```
Comment (mihaic) [inline brainiak/factoranalysis/htfa.py]: We should remove the default from the docstring too.

As an aside, having to change the docstring now is a good reason to not put the default in the docstring in the first place, given that it is visible in the signature and Sphinx adds it to the HTML docs.

File: brainiak/factoranalysis/htfa.py (hunk 1 above)
Comment (xiazhu) [inline brainiak/factoranalysis/htfa.py]: R is not required for construct HTFA. It is a parameter to fit() function.

File: brainiak/factoranalysis/htfa.py (hunk 2 above)
Comment (xiazhu) [inline brainiak/factoranalysis/htfa.py]: updated with new commit.

**Comment by mihaic** [comment]
Thank you for the quick PR, @xiazhu.

As another aside, the test giving the last error you were seeing is a good example of a too specialized test.

**Pull request merged**